app/javascript/shared/components/tags.jsx

```diff
@@ -253,23 +257,26 @@ class Tags extends Component {
 
     if (
       component.selected.length === this.props.maxTags &&
-      e.keyCode === KEYS.COMMA
+      e.keyCode === ACTION_KEYS.COMMA
```

Comment:
I feel like there is some way to refactor a significant amount of this conditional logic, but perhaps that is out of the scope of this PR.

Reply:
I would tend to agree that it's out of the scope of this PR - I think it makes sense to keep a separation of concerns. 

That said, I'd be happy to submit a followup PR aiming to simplify a lot of this code if you feel it's worthwhile!

Reply:
@jacobherrington Np! I went ahead and added a validation on `Tag.name` with some specs [here](https://github.com/thepracticaldev/dev.to/pull/4283/commits/23ac1cc4996be57b4b2436ade95822244ef7a532)